modules/ossm-migrating-from-smcp-defined-gateways-to-gateway-injection.adoc

```diff
@@ -0,0 +1,142 @@
+    name: istio-ingressgateway
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy <3>
+metadata:
+  name: gatewayingress
+  namespace: istio-system
+spec:
+  podSelector:
+    matchLabels:
+      istio: ingressgateway
+  ingress:
+    - {}
+  policyTypes:
+    - Ingress
+---- 
+<1> The gateway injection deployment and all supporting resources should be deployed in the same namespace as the SMCP-defined gateway. 
+<2> Ensure that the labels specified in the pod template include all of the label selectors specified in the Service associated with the existing SMCP-defined gateway.
+<3> Grant access to the new gateway from outside the cluster. This access is required whenever the `spec.security.manageNetworkPolicy` of the `ServiceMeshControlPlane` resource is set to `true`, which is the default setting.
+
+. Verify that the new gateway deployment is successfully handling requests. 
++
+Ensure that the new gateway deployment is successfully handling requests. If access logging was configured in the `ServiceMeshControlPlane` resource view the access logs of the new gateway deployment to confirm the behavior.
+
+. Scale down the old deployment and scale up the new deployment. 
++
+You may gradually shift traffic from the old gateway deployment to the new gateway deployment by running the `oc scale` command at the command line:
++
+[source,terminal]
+----
+$ oc scale -n istio-system deployment/<gateway_deployment> --replicas <new_number_of_replicas>
+----
++
+As you scale, increase the number of replicas for the new gateway deployment and decrease the number of replicas for the old gateway deployment. Continue scaling until the new gateway deployment handles all traffic to the gateway Service. 
+
+. Remove the `app.kubernetes.io/managed-by` label from the gateway Service resource.
++
+Removing the label prevents the Service from being deleted when the gateway is disabled in the `ServiceMeshControlPlane` resource. Run the following command at the CLI:
++
+[source,terminal]
+----
+$ oc label service -n istio-system istio-ingressgateway app.kubernetes.io/managed-by-
+----
+
+. Remove the `ownerReferences` object from the gateway Service resource.
++
+Removing this object prevents the Service from being garbage collected when the `ServiceMeshControlPlane` resource is deleted.
++
+[source,terminal]
+----
+$ oc patch service -n istio-system istio-ingressgateway --type='json' -p='[{"op": "remove", "path": "/metadata/ownerReferences"}]'
+----
+
+. Disable the old gateway deployment that was managed by the `ServiceMeshControlPlane` resource by running the following command at the CLI:
++
+[source,terminal]
+----
+$ oc patch smcp -n istio-system <smcp_name> --type='json' -p='[{"op": "replace", "path": "/spec/gateways/ingress/enabled", "value": false}]'
+----
++
+When the old ingress gateway Service resource is disabled it is not deleted. You may save this Service resource to a file and manage it alongside the new gateway injection resources.
```

Comment:
Just checking - is this info something you think could be wrapped in a NOTE?

Reply:
done